easybuild/easyconfigs/w/WRF/WRF-4.0.1-intel-2018b-dmpar.eb

```diff
@@ -0,0 +1,40 @@
+name = 'WRF'
+version = '4.0.1'
+
+homepage = 'http://www.wrf-model.org'
+description = """The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale
+ numerical weather prediction system designed to serve both operational forecasting and atmospheric
+ research needs."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+toolchainopts = {'opt': False}  # don't use agressive optimization, stick to -O2
+
+source_urls = ['https://github.com/wrf-model/WRF/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'WRF_parallel_build_fix.patch',
+    'WRF-%(version)s_netCDF-Fortran_separate_path.patch',
+    # 'WRF-%(version)s_known_problems.patch',  # no known problems as of 2018-10-19
```

Comment:
You can just drop this commented patch.

Reply:
@andreas-h  +1, please remove this line?

Reply:
done.